Trimble.Modus.Components/Controls/TMRadioButton/TMRadioButton.cs

```diff
@@ -119,12 +130,32 @@ public TMRadioButton()
             };
             _tapGesture.Tapped += OnTapGestureTapped;
             GestureRecognizers.Add(_tapGesture);
-            Margin = new Thickness(0, 0, 5, 5);
+            Margin = new Thickness(0, 0, 5, 5);
+
+            this.SetDynamicResource(IconTintColorProperty, "RadioButtonColor");
         }
 
         #endregion
 
         #region Private Methods
+        /// <summary>
+        /// Update Tint color based on theme
+        /// </summary>
+        private static void OnIconTintColorPropertyChanged(BindableObject bindable, object oldValue, object newValue)
+        {
+            if (bindable is TMRadioButton tmRadioButton && DeviceInfo.Platform != DevicePlatform.WinUI)
```

Comment:
`DeviceInfo.Platform != DevicePlatform.WinUI`-> I assume this is to avoid duplicate icons issue in Windows. But this means that the dark theme won't work in Windows, right?
@BAL4-R Can you add a FIXME comment?
@ElishaSamPeterPrabhu Can you verify this in Windows once?

Reply:
![image](https://github.com/trimble-oss/modus-mobile-maui-components/assets/122255626/d7502dac-2a89-426c-b049-f1ba074fe15b)
Dark theme is not applied with this condition